frontend/dashboard/app/App.test.tsx

```diff
@@ -10,9 +10,18 @@ import { QueryKey } from 'app-shared/types/QueryKey';
 import { user as userMock } from 'app-shared/mocks/mocks';
 import type { QueryClient } from '@tanstack/react-query';
 import userEvent from '@testing-library/user-event';
+import type { Organization } from 'app-shared/types/Organization';
```

Comment:
Nitpick:
I think it was mentioned somewhere that we should spell `Organization` and similar words as `Organisation`?
```suggestion
import type { Organisation } from 'app-shared/types/Organisation';
```

Reply:
I agree that we should choose British spelling, but this type is used in _lots_ of places. I suggest we fix that in a separate pull request.